AUTHORS.rst

```diff
@@ -39,7 +39,6 @@ Contributors
 - `@Eidhagen <https://github.com/Eidhagen>`_ | `contributions <https://github.com/ericmjl/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Aclosed+mentions%3AEidhagen>`_
 - `@mdini <https://github.com/mdini>`_ | `contributions <https://github.com/ericmjl/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Aclosed+mentions%3Amdini>`_
 - `@kimt33 <https://github.com/kimt33>`_ | `contributions <https://github.com/ericmjl/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Aclosed+mentions%3Akimt33>`_
-- @jack-kessler-88 | user no longer found
```

Comment:
Though @jack-kessler-88 is not found, they should remain on the contributors list, because Jack as a person does exist :smile:.

Reply:
This may have just been an error when merging. I'll just add it back.